### ISSUE TITLE
Shell integration error has bad contrast in Default Dark

### ISSUE THREAD

**Issue by lramos15** [microsoft/vscode#144269]
The error codicon shown when a command is invalid is very hard to see and fails accessibility contrast checks.

Tested by pulling the colors from dev tools and plugging them into contrast checker https://contrastchecker.com/?c=994411&b=1E1E1E

**Comment by meganrogge** [comment]
Yes we're aware of this and are using the same colors that the SCM uses for decorations. 

@misolori didn't you mention that you updated the SCM decoration colors yesterday?

**Comment by misolori** [comment]
The SCM colors are updated in main https://github.com/microsoft/vscode/issues/142809#issuecomment-1055892358 so they'll make it in the next Insiders (1.66)

However I noticed that the colors are not inheriting the same colors from the decorators?

![CleanShot 2022-03-02 at 11 38 59@2x](https://user-images.githubusercontent.com/35271042/156436928-805a84cb-2135-48ed-961b-0d2ee0a46894.png)

**Comment by misolori** [comment]
Yea it looks like the colors aren't properly inheriting the scm colors:

https://github.com/microsoft/vscode/blob/35f8ba6f6678a818f8d3e994643bd9f167c9c422/src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts#L41-L45

So they should be:

```ts
export const TERMINAL_COMMAND_DECORATION_ERROR_BACKGROUND_COLOR = registerColor('terminalCommandDecoration.errorBackground', {
	dark: diffRemovedLineGutter,
	light: diffRemovedLineGutter,
	hc: diffRemovedLineGutter
}, nls.localize('terminalCommandDecoration.errorBackground', 'The terminal command decoration background color for error commands.'));
```

Let me know if you want me to setup a PR and can happily do that

**Comment by Tyriar** [comment]
@misolori they don't inherit anymore because we want to avoid the terminal depending on the editor and there was a layer breaker when we did do that. Currently they're copied until we have that base color idea I was mentioning earlier, not sure if that's tracked.

Taking off Feb as it's a minor visual problem with a workaround on an experimental feature.

**Comment by misolori** [comment]
I see, that makes sense and agree that at least updating the new default colors will work better for now until we have a better color system.
